op-node/rollup/derive/channel_test.go

```diff
@@ -145,11 +145,15 @@ func TestFrameValidity(t *testing.T) {
 
 func TestBatchReader(t *testing.T) {
 	rng := rand.New(rand.NewSource(0x543331))
-	singularBatch := RandomSingularBatch(rng, 20, big.NewInt(333))
-	batchDataInput := NewBatchData(singularBatch)
+	singularBatch1 := RandomSingularBatch(rng, 20, big.NewInt(333))
+	batchDataInput1 := NewBatchData(singularBatch1)
+	singularBatch2 := RandomSingularBatch(rng, 20, big.NewInt(333))
+	batchDataInput2 := NewBatchData(singularBatch2)
 
 	encodedBatch := new(bytes.Buffer)
-	err := batchDataInput.EncodeRLP(encodedBatch)
+	err := batchDataInput1.EncodeRLP(encodedBatch)
+	require.NoError(t, err)
+	err = batchDataInput2.EncodeRLP(encodedBatch)
 	require.NoError(t, err)
```

Comment:
Would prefer to work on a slice with two batches and datas, instead of copy-pasting. Could then also easily do 3 batches.

Reply:
Good point, done in a11dbba.